HISTORY.md

```diff
@@ -1,3 +1,19 @@
+# 5.0 / 2020-03-17
+
+This is a major new release that substantially cleans up and improves the codebase. It also introduces many new features, bugfixes, and removing major limitations. See [here](https://github.com/gollum/gollum/wiki/5.0-release-notes) for a list of changes.
+
+**Note**: due to changes to the way in which Gollum handles filenames, you may have to change some links in your wiki when migrating from gollum 4.x. See the [release notes](https://github.com/gollum/gollum/wiki/5.0-release-notes#migrating-your-wiki) for more details. You may be able to use the `bin/gollum-migrate-tags` script to accomplish this. Also see the `--lenient-tag-lookup` option.
```

Comment:
"Also see the `--lenient-tag-lookup` option." -> Also see the `--lenient-tag-lookup` option for making tag lookup backwards compatible with 4.x. (though note that this will slow down large wikis with many tags). (or something to that effect).